hack/kubeadminkubeconfig/get.go

```diff
@@ -64,7 +66,17 @@ func main() {
 		log.Fatalf("usage: %s resourceid\n", os.Args[0])
 	}
 
-	err := writeKubeconfig(ctx, os.Args[1])
+	timeoutCtx, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
+
+	err := wait.PollImmediateUntil(2*time.Second, func() (bool, error) {
```

Comment:
this will mask transient errors against the RP, and I'm not sure that's a great idea.  I think solely wrapping the calls to getTokenURLFromConsoleURL and getAuthorizedToken would be tighter.